Source/ThirdParty/GltfPipeline/updateVersion.js

```diff
@@ -508,26 +508,6 @@ define([
                     delete image.extensions;
                 }
             }
-            if (defined(image.extras)) {
-                var compressedImages = image.extras.compressedImage3DTiles;
-                for (var type in compressedImages) {
-                    if (compressedImages.hasOwnProperty(type)) {
-                        var compressedImage = compressedImages[type];
-                        var compressedExtensions = compressedImage.extensions;
-                        if (defined(compressedExtensions)) {
-                            var compressedBinaryGltf = compressedExtensions.KHR_binary_glTF;
-                            if (defined(compressedBinaryGltf)) {
-                                compressedImage.bufferView = globalMapping.bufferViews[compressedBinaryGltf.bufferView];
-                                compressedImage.mimeType = compressedBinaryGltf.mimeType;
-                                delete compressedExtensions.KHR_binary_glTF;
-                            }
-                            if (Object.keys(extensions).length === 0) {
```

Comment:
@lilleyse also, this looks like a copy-paste error.  should this be `compressedExtensions`?

Reply:
Oh, yeah I think so.